### DISCUSSION TITLE
Mass/Charge of precursors

### DISCUSSION THREAD

**Discussion by jeewan-rijal** [vdemichev/DiaNN#394] [Q&A]
Hello Vadim
I am processing DIA-Pasef data of different sample types.
I would like to get the mass/charge of the precursors quantified that are present in the main report. I see the charge but not the mass of each precursor in the report. Is it possible to obtain this value somehow or did I miss it?
Thank you very much.

**Comment by vdemichev** [comment]
Hi, unfortunately DIA-NN does not output empirical precursor m/z in the current version.

Vadim

**Reply by vdemichev** [reply]
You can get the reference information about the precursor by matching the report to the spectral library though.

**Reply by t-jumel** [reply]
Hello, it would be indeed nice and will likely be requested more often in the future
to have the Precursor.Mz from the main report included in the pr_matrix.
(I do not know if diaPASEF results look different.)
Best, tobi

**Reply by jeewan-rijal** [reply]
Thanks a lot for your very quick answer.
It would be great to see this value in the report in future versions.
Best, Jeewan

**Comment by ManuTheFil** [comment]
Dear Vadim,
I see observed m/z information is still not in the report with DIANN release 1.8.1
Is there a plan in order to add it in the future ?
Or did I miss how to get this information ?
Kindly yours,
Manuel Chapelle from Bruker France
